src/main/java/nl/unimaas/bigcat/wikipathways/nanopubs/PubMed.java

```diff
@@ -0,0 +1,96 @@
+package nl.unimaas.bigcat.wikipathways.nanopubs;

```

Comment:
Please add a copyright/license header here like in https://github.com/wikipathways/nanopublications/blob/master/src/main/java/nl/unimaas/bigcat/wikipathways/nanopubs/ComplexesPubs.java